tests/cylc-poll/12-reverse-state.t

```diff
@@ -1,32 +0,0 @@
-#!/bin/bash
-# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
-# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
-#
-# This program is free software: you can redistribute it and/or modify
-# it under the terms of the GNU General Public License as published by
-# the Free Software Foundation, either version 3 of the License, or
-# (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#-------------------------------------------------------------------------------
-
-# Run a test suite to check that a seemingly-failed task that is actually still
-# running can be returned to the running state by polling, and other aspects of
-# PR #2396 on GitHub.
```

Comment:
Behaviour change?

Reply:
Change documented here: https://gist.github.com/hjoliver/ef10394a3161713dbfad45d68238d079#cylc-poll

Reply:
(But yes we'll need to document these "behaviour changes" well!)